ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -683,3 +684,50 @@ def test_jax_numpy_nanvar(
         atol=1e-3,
         rtol=1e-3,
     )
+
+
+# std
+@with_unsupported_dtypes({"1.11.0 and below": ("bfloat16")}, "jax")
```

Comment:
This decorator is supposed to be added to the function, not the test.

Reply:
ok